src/demo/app/charts/charts-category-demo.component.ts

```diff
@@ -0,0 +1,86 @@
+import { Component } from '@angular/core';
+import { CategoryChartModel } from 'ngx-prx-styleguide';
+
+@Component({
+  selector: 'charts-category-demo',
+  template: `
+    <section class="main demo">
+      <h1>CategoryChart</h1>
+      <p>
+        A Category Chart is where data is given categories as labels on the x axis.
+      </p>
+      <dl>
+        <dt>module</dt><dd><code>ChartsModule</code></dd>
+        <dt>selector</dt><dd><code>prx-category-chart</code></dd>
+      </dl>
+      <ul>
+        <li>
+          <code>@Input() data: CategoryChartModel[]</code>
+          - an array of CategoryChartModel, i.e. <code>[&#123; value: number, label: string; &#125;]</code>
+        </li>
+        <li>
+          <code>@Input() color: string</code>
+          - hex color code for all bars, default <code>'#0089bd'</code>
+        </li>
+        <li>
+          <code>@Input() rotated: boolean</code>
+          - if <code>true</code>, the axis is rotated, default is <code>true</code>
+        </li>
+        <li>
+          <code>@Input() dataLabel: string</code>
+          - general label describing to bar amount, displayed in hover with amount, default is <code>'amount'</code>
+        </li>
+      </ul>
+      
+      <aside>
+        <h2>Bar Chart</h2>
+        <p>
+          A Bar Chart is used to show comparisons among categories. Bars will be ordered in the order given.
+        </p>
+        Usage:
+        <pre class="code">
+          &lt;prx-category-chart [data]="barData" [rotated]="noRotate"&gt;
```

Comment:
I understand that noRotate is "false" here. Which means we will not be rotated.  Not a big deal, but we could swap in the literal for the clarity?

Reply:
meh, you can't do that with false, i.e. `rotated="false"` would evaluate to the string "false", mildly annoying